### PR TITLE
Updated the readme to fix spelling and grammar mistakes

### PR DESCRIPTION
**Pull request by RazersTv**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,11 +10,11 @@
...
 - Define distance on a per-option or overall basis when triggering qtarget function exports
-- Ability to redefine or remove options, and add new options without replacing old ones
-- Update option list when moving towards or away from a target with variable distances on their options
-- Support for entity bones, with builtin tables for opening vehicle doors
+- Ability to redefine or remove options and add new options without replacing old ones
+- Update the option list when moving towards or away from a target with variable distances on their options
+- Support for entity bones with built-in tables for opening vehicle doors
 - Support checking for job, items, or specific entities
-- Utilise the `canInteract` function for advanced checks to show or hide an option based on any trigger
+- Utilize the `canInteract` function for advanced checks to show or hide an option based on any trigger
```
Comment (BerkieBb) [inline README.md:17]: Utilise is with an s in the English language, Utilize is what Americans use

Comment (RazersTv) [reply README.md:17]: Thats what I thought to but then I read this. https://grammarist.com/grammar/use-vs-utilize/

Comment (RazersTv) [reply README.md:17]: utilise appears to just be the British spelling where as the rest of the world it is utilize

Comment (RazersTv) [reply README.md:17]: Thats according to websters dictionry

Comment (BerkieBb) [reply README.md:17]: > utilise appears to just be the British spelling where as the rest of the world it is utilize

That's the language the readme goes by

Comment (RazersTv) [reply README.md:17]: oh...

Comment (RazersTv) [reply README.md:17]: welp

**Pull request merged**